### DISCUSSION TITLE
Sending data to ibbroker

### DISCUSSION THREAD

**Discussion by dannyv1101** [awawa-dev/HyperHDR#993] [General]
Hi, is there a way to create a virtuell Light and send the rgb value to iobroker maybe with mqtt or something else? 

I have some rgb light near my TV and I want to controll them by hyperHDR. All my lights are conected to iobroker with zigbee2mqtt.

Thanks, Danny

**Comment by awawa-dev** [comment]
Hi
udpraw driver sends RGB colors using UDP. You can use it as a template to create your own LED device (not necessarily based on UDP).

**Comment by dannyv1101** [comment]
Ok thank you, I will try it today.

Do you know how to set it up in iobroker? I want to try to use the multicast adapter.

**Comment by awawa-dev** [comment]
Unfortunately no, I don't use iobroker.
